windows/x_walletconfig_w.cpp

```diff
@@ -386,6 +400,10 @@ void WalletConfig::on_restoreDefault_clicked()
 
     updateLogsStateUI(true);
 
+    updateAutoStartStateUI(true, true);
+
+    updateAutoLogoutStateUI(3 * 60);
```

Comment:
Please make larger default value. Currently default for config is 15 minutes. You can make it 20.